xml/deployment_pxe.xml

```diff
@@ -290,24 +290,59 @@ DHCLIENT6_CLIENT_ID=<replaceable>00:03:52:54:00:02:c2:67</replaceable></screen>
    <title>Installing Files for Boot</title>
    <para>
     &productname; provides the required files for booting via a PXE on BIOS or
-    UEFI machines in an RPM. Install the files on the machine running the TFTP server:
+    UEFI machines. The following hardware architectures are supported:
+   </para>
+
+   <itemizedlist>
+    <listitem>
+     <para>
+      &x86-64;
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      &aarch64;
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      &power;
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      &zseries;
+     </para>
+    </listitem>
+   </itemizedlist>
+
+   <para>
+    Files required to boot from a specific hardware architecture are included
+    in a RPM package. Install it on the machine running the TFTP server:
    </para>
    <screen>&prompt.sudo;<command>zypper in tftpboot-installation-SLE-<replaceable>OS_VERSION</replaceable>-<replaceable>ARCHITECTURE</replaceable></command></screen>
    <para>
     Replace <replaceable>OS_VERSION</replaceable> with the version of your
-    &productname; installation, for example <replaceable>SLE-15-SP1-x86_64</replaceable> 
-    with the architecture of your
-    system, for example <literal>x86_64</literal>. Run
-    <command>zypper se tftpboot</command> to search for all available
-    versions and architectures.
+    &productname; installation, for example
```

Comment:
I think a few words were lost here (already, in the older text) and what is left is garbled.

> Replace &lt;replaceable&gt;OS_VERSION&lt;/replaceable&gt; with the version of your &productname; installation,
> for example &lt;replaceable&gt;SLE-15-SP1-x86_64&lt;/replaceable&gt; with the architecture of your system, for 
> example &lt;literal&gt;x86_64&lt;/literal&gt;.

I think this should read something like the following:

> Replace &lt;replaceable&gt;OS_VERSION&lt;/replaceable&gt; with the version number of your &productname; 
> installation, for example &lt;replaceable&gt;15-SP1&lt;/replaceable&gt;, and replace
> &lt;replaceable&gt;ARCHITECTURE&lt;/replaceable&gt; with the architecture of your system, for example 
> &lt;literal&gt;x86_64&lt;/literal&gt;. So the resulting text would look like this:
> &lt;filename&gt;tftpboot-installation-SLE-15-SP1-X86_64&lt;/filename&gt;.

I also note that there is a mixture of lower-case and upper-case in the example (x86-64 **and** X86-64) but I am not sure which is correct.